Move functions related to random number generations from math module (#1)
ballerina-platform/module-ballerina-random: README.md

Ballerina Random Library
===================

  [![Build](https://github.com/ballerina-platform/module-ballerina-random/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-random/actions?query=workflow%3ABuild)
  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-random.svg)](https://github.com/ballerina-platform/module-ballerina-random/commits/main)
  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/random.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Frandom)
  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)

> The Random library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.

Reviewer: ```suggestion
The Random library is one of the standard library modules of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```

This module provides functions related to random number generation.

## Issues and Projects

> Issues and Projects tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).

Reviewer: ```suggestion
The **Issues** and **Projects** tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc., go to the Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

> This repository only contains the source code for the module.

Reviewer: ```suggestion
This repository contains only the source code of the module.
```
Shall we put this sentence at the beginning of the "## Issues and Projects" topic?

## Building from the Source

### Setting Up the Prerequisites

1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
   
   * [OpenJDK](https://adoptopenjdk.net/)
   
        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
     

> 2. Export Github Personal access token with read package permissions as follows,

Reviewer: ```suggestion
2. Export your Github Personal access token with the read package permissions as follows.
```

Ballerina Random Library
===================

  [![Build](https://github.com/ballerina-platform/module-ballerina-random/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-random/actions?query=workflow%3ABuild)
  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-random.svg)](https://github.com/ballerina-platform/module-ballerina-random/commits/main)
  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/random.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Frandom)
  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)

The Random library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.

This module provides functions related to random number generation.

> For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/).

Reviewer 2: ```suggestion
```

Reviewer 2: Shall we remove this. since we don't have a BBE to point. will add this when we add a BBE

Author: Ack. Temporarily removed.
